frontend/ui/data/creoptions.lua

```diff
@@ -65,6 +171,9 @@ local CreOptions = {
                     DCREREADER_CONFIG_LINE_SPACE_PERCENT_MEDIUM,
                     DCREREADER_CONFIG_LINE_SPACE_PERCENT_LARGE,
                 },
+                suffix = "%",
+                true_values = true,
+                name_text_hold_callback = showValues,
```

Comment:
I was first going to suggest to call these `name_text_suffix` and `name_text_true_values` so we know quickly that they are just used by `name_text_hold_callback` (and we don't go wondering if they are used by configdialog), but it would look ugly, and may be these `suffix` and `true_values` could be used later for other things.
May be you could just move them below `name_text_hold_callback`, and just add a comment on their right `-- used by showValues` ?
Looks clean overall :)